jvm/src/main/scala/org/apache/spark/ml/rapids/RapidsLogisticRegressionModel.scala

```diff
@@ -50,4 +56,44 @@ class RapidsLogisticRegressionModel(override val uid: String,
     newModel
   }
 
+  override def write: MLWriter =
+    new RapidsLogisticRegressionModel.RapidsLogisticRegressionModelWriter(this)
+}
+
+object RapidsLogisticRegressionModel extends MLReadable[RapidsLogisticRegressionModel] {
+
+  override def read: MLReader[RapidsLogisticRegressionModel] = new RapidsLogisticRegressionModelReader
+
+  override def load(path: String): RapidsLogisticRegressionModel = super.load(path)
+
+  private class RapidsLogisticRegressionModelWriter(instance: RapidsLogisticRegressionModel)
```

Comment:
Looks like saved model won't be compatible with original spark-rapids-ml format.  Not sure we need that.   

Also, looks like it **is compatible** with spark mllib if plugin is disabled, it can be read in as spark mllib model.  Am I seeing that correctly?

Reply:
Yeah, You're right. Let me fix it in a followup.